{
  "comments": [
    {
      "unresolved": true,
      "key": {
        "uuid": "5a596190_abe0c4cb",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 2
      },
      "lineNbr": 0,
      "author": {
        "id": 1000011
      },
      "writtenOn": "2024-07-16T18:20:58Z",
      "side": 1,
      "message": "Would it instead be better to have launch_daemon update environments somehow? All applications started from Tracker will have this same problem.",
      "revId": "81a5ea81432866f914bcf4e205006200753795b8",
      "serverId": "40b9299a-d8a8-485d-9b01-e6d3f45eefb5"
    }
  ]
}